src/rulesets/Base/sort.jl

```diff
@@ -26,7 +26,7 @@ function rrule(::typeof(sort), xs::AbstractVector; kwargs...)
             return Δxs
         end
 
-        Δxs = InplaceableThunk(@thunk(sort_add!(zero(Δys))), sort_add!)
+        Δxs = InplaceableThunk(@thunk(sort_add!(zero(xs))), sort_add!)
```

Comment:
This seems wrong.
The zero of the primal doesn't have to exist.
(If it does, it is probably also the zero of the differential)

Consider `xs::Vector{DateTime}`
then `Δys` is either a `Vector{<:Period}` or a `Vector{<:Tangent{DateTime}}`

In this case it is bugged.
https://github.com/JuliaLang/julia/issues/41348
but if it was a a `Vector{<:Period}` it wouldn't be